chat-plugins/economy.js

```diff
@@ -122,6 +122,7 @@ exports.commands = {
 
 		Economy.writeMoney(targetUser, amount);
 		this.sendReply(Tools.escapeHTML(targetUser) + " has received " + amount + ((amount === 1) ? " buck." : " bucks."));
+		if (Users.getExact(targetUser) && Users.getExact(targetUser)) Users.getExact(targetUser).popup("|modal||html|You have been given " + amount + ((amount === 1) ? " buck" : " bucks") + " by <b><font color='" + Wisp.hashColor(user.name) + "'>" + user.name + "</font></b>" )
```

Comment:
It should be <code>Users.getExact(targetUser) && Users.getExact(targetUser).connected</code>